tests/unit/utils/env/index.test.ts

```diff
@@ -65,24 +65,24 @@ test("should filter, sort, and format Envelope's response correctly", () => {
   const envelopeItems = [
```

Comment:
If you export the `EnvelopeItem` type, you can:

```suggestion
  const envelopeItems: EnvelopeItem[] = [
```

Which eliminates the need to assert strings as `const`s.